conf/cut_release.sh

```diff
@@ -187,6 +192,20 @@ check_branch_dirty() {
     fi
 }
 
+check_npm_registry() {
+    if [[ ! $(npm config get registry) =~ (${YARN_PUBLIC_REGISTRY_REGEX}|${NPM_PUBLIC_REGISTRY_REGEX}) ]] ; then
+        printf "${red}Not pointing at the right npm registry!${end}"
```

Comment:
Maybe print a message to update .npmrc with the url we want them to use?

Reply:
Sounds good

Reply:
Yes, let's add in a known step to address the issue in each case. Here, if they're pointed at the wrong place, they'll probably need to:

* move their current config and `npm login`
* or, do some custom commenting in their config file

I'd lean toward encouraging something like the former to avoid user error